Source/NSStoryboard.m

```diff
@@ -0,0 +1,975 @@
+                  [ss _setKind: kind];
+                  [ss _setRelationship: rel];
+                  
+                  // Add to maptable...
+                  [mapTable setObject: ss
+                               forKey: identifier];
+                  
+                } // only process segue objects...
+            } // iterate over objects in each set of connections
+        } // iterate over connection objs
+
+      // Add to cache...
+      [_documentsMap setObject: mapTable
+                        forKey: uuidString];
+      
+    }  // if connections > 0
+  return mapTable;
+}
+
+// Private instance methods...
+- (id) initWithName: (NSStoryboardName)name
+             bundle: (NSBundle *)bundle
+{
+  self = [super init];
+  if (self != nil)
+    {
+      NSString *path = [bundle pathForResource: name
+                                        ofType: @"storyboard"];
+      NSData *data = [NSData dataWithContentsOfFile: path];
+      
+      NSXMLDocument *storyboardXml = [[NSXMLDocument alloc] initWithData: data
+                                                                 options: 0
+                                                                   error: NULL];
+      [self _processStoryboard: storyboardXml];
+      RELEASE(storyboardXml);
+    }
+  return self;
+}
+
+// Class methods...
++ (void) _setMainStoryboard: (NSStoryboard *)storyboard  // private, only called from NSApplicationMain()
+{
+  if (__mainStoryboard == nil)
+    {
+      ASSIGN(__mainStoryboard, storyboard);
+    }
+}
+
++ (NSStoryboard *) mainStoryboard // 10.13
+{
+  return __mainStoryboard;
+}
+
++ (instancetype) storyboardWithName: (NSStoryboardName)name
+                             bundle: (NSBundle *)bundle
+{
+  return AUTORELEASE([[NSStoryboard alloc] initWithName: name
+                                                 bundle: bundle]);
+}
+
+// Instance methods...
+- (void) dealloc
+{
+  RELEASE(_initialViewControllerId);
+  RELEASE(_applicationSceneId);
+  RELEASE(_scenesMap);
+  RELEASE(_controllerMap);
+  RELEASE(_documentsMap);
+  [super dealloc];
+}
+
+- (void) _instantiateApplicationScene
+{
+  NSDictionary	*table;
+
+  table = [NSDictionary dictionaryWithObject: NSApp
+                                      forKey: NSNibOwner];
+
+  NSXMLDocument *xml = [_scenesMap objectForKey: _applicationSceneId];
+  NSData *xmlData = [xml XMLData];
+  GSModelLoader *loader = [GSModelLoaderFactory modelLoaderForFileType: @"xib"];
+  BOOL success = [loader loadModelData: xmlData
+                     externalNameTable: table
+                              withZone: [self zone]];
+  if (!success)
+    {
+      NSLog(@"Unabled to load Application scene");
+    }
+}
+
+- (id) instantiateInitialController
+{
+  return [self instantiateControllerWithIdentifier: _initialViewControllerId];
+}
+
+- (id) instantiateInitialControllerWithCreator: (NSStoryboardControllerCreator)block // 10.15
+{
+  id controller = [self instantiateInitialController];
```

Comment:
Normally we call the more general method from the specific one, here you do it the other way around

Reply:
Fixing this now.

Reply:
Done.